app/controllers/idv/personal_key_controller.rb

```diff
@@ -30,6 +30,7 @@ def update
         fraud_review_pending: fraud_review_pending?,
         fraud_rejection: fraud_rejection?,
       )
+      idv_session.personal_key = nil
```

Comment:
Do we have test coverage for this? I guess I'm a little lost trying to understand what's happening here with the personal key in the session. It looks like we move it into `user_session` ? Alternatively, could we update the logic of `def personal_key` to pull from `user_session` if it's expected to be there?

Reply:
I'm not sure why it was set to nil in `show` before, so I figured I'd set it to nil in `update` now. Maybe pulling from user_session would be cleaner. I'll try that out.

Reply:
Update: It breaks the specs. I'm inclined to do the bug fix and add a ticket for cleaning up this code overall.

Reply:
I think it is reasonable to set the personal key to `nil` here. We do not want to hang onto the personal key longer than we need to.

Reply:
Ah, I see. We are moving the personal key into the user session and then presumably not reading it ever. This looks like it might be an artifact or copy/paste noise from when the personal key page here and the personal key page in the account page were the same. I don't think we want to set the personal key in the user sessions.

I'm with Sonia on cleaning this up so the personal key does not get written to the user session.